site/data/theme-colors.yml

```diff
@@ -20,3 +20,4 @@
   contrast_color: dark
 - name: dark
   hex: "#000"
+  contrast_color: white
```

Comment:
Need to introduce it for `secondary` to be consistent ?

Reply:
Removed entirely `contrast_color` from this file via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1757/commits/7b9e84681cfa9f1eabbfd261ad6770648f06b1bd (+ https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1757/commits/4b7d435129cc3eac7bd68536e6e7162cc4469a16 because I'm stupid 😄) because it's not used in Boosted